branchpro/_incidence_number_plot.py

```diff
@@ -0,0 +1,81 @@
+#
+# IncidenceNumberPlot Class
+#
+# This file is part of BRANCHPRO
+# (https://github.com/SABS-R3-Epidemiology/branchpro.git) which is released
+# under the BSD 3-clause license. See accompanying LICENSE.md for copyright
+# notice and full license details.
+#
+
+import pandas as pd
+import plotly.graph_objs as go
+
+
+class IncidenceNumberPlot():
+    """IncidenceNumberPlot Class
+    Stores the main figure for the Dash app.
+    """
+    def __init__(self):
+        self._figure = go.Figure()
```

Comment:
I would leave figure as a public variable, we may want to access it from outside the class